Fixed Overflow of Navbar.

Phoenix6296 opened the pull request:
Fixes #4  Added some media queries in the style.css for fixing the overflow problem of Navbar.

Phoenix6296 committed:
Fixed Overflow of Navbar.

Mitrajit commented:
The Sharenetic title seems even smaller than the buttons. Let's do something else to make the proportions correct.
The download and upload icons seem too spaced.
![image](https://user-images.githubusercontent.com/55190574/164885335-67856175-fdea-47d2-8302-6bfdac0cc932.png)
TODO
- [x] Correct the 'Sharenetic' title proportions
- [x] Reduce the spaces between upload and download icon
- [x] Also try Samsung fold and try to correct the overflow

Phoenix6296 committed:
Changes Made:
i. Made the Navbar Responsive and removed all overflows.
ii. Interchanged the button positions of connect and copy id.
iii. Reciepient Id of Text Box of container is made responsive.

Phoenix6296 committed:
Icons size of upload and download were corrected.

Phoenix6296 committed:
Icons size problem for smaller devices are fixed.

Mitrajit commented on index.html:
```diff
@@ -18,30 +18,38 @@
+  <div class="container box p-4 whole-body">
+    <div class="input-group pb-3 head-place">
+      <div class="recipent">
+        <input type="text" class="form-control" id="recipent-input" placeholder="Recipient's ID"
```
The ID for the textbox was changed. Please don't change any ID as they are crucial for the javascript to work.

Phoenix6296 replied:
Done Dona Done :)

Phoenix6296 committed:
Updated the ID of the input.

Mitrajit committed:
Text box and copy put in the same group

Mitrajit commented:
Thanks 👍🏻
